base/operators.jl

```diff
@@ -871,9 +871,37 @@ julia> fs = [
 julia> ∘(fs...)(3)
 3.0
 ```
+See also [ComposedFunction](@ref).
```

Comment:
```suggestion
See also [Base.ComposedFunction](@ref).
```

I think this might be required for building documentation? Let's see...